README.md

```diff
@@ -2,3 +2,54 @@
 [![npm last version](https://img.shields.io/npm/v/@nextcloud/files.svg?style=flat-square)](https://www.npmjs.com/package/@nextcloud/files) [![Code coverage](https://img.shields.io/codecov/c/github/nextcloud-libraries/nextcloud-files?style=flat-square)](https://app.codecov.io/gh/nextcloud-libraries/nextcloud-files) [![Project documentation](https://img.shields.io/badge/documentation-online-blue?style=flat-square)](https://nextcloud.github.io/nextcloud-files/)
 
 Nextcloud Files helpers for Nextcloud apps and libraries
+
+## Usage example
+
+### Using WebDAV to query favorite nodes
+
+```ts
+import { davGetClient, davRootPath, getFavoriteNodes } from '@nextcloud/files'
+
+const client = davGetClient()
+// query favorites for the root folder (meaning all favorites)
+const favorites = await getFavoriteNodes(client)
+// which is the same as writing:
+const favorites = await getFavoriteNodes(client, '/', davRootPath)
```

Comment:
Why does getFavoriteNodes have a root path?
We are always requesting `SEARCH` on the default root, no?

Reply:
completely forgot this was a REPORT, make sense then :)